src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball extends ColorSupplier {
```

Comment:
`Ball` is not meant to be an extension for `ColorSupplier` - those are two separate entities in the context of this application.
Ball IS NOT a color supplier.